### DISCUSSION TITLE
Improve Dev > Statistics > Adjust a statistic

### DISCUSSION THREAD

**Discussion by mattmeye** [home-assistant/frontend#15123] [Other feature requests]
Hi,
one of my senors generated wrong data over a couple of days. I want to remove the data manually but the UI/UX is very uncomfortable for doing this.

Problem: 

So I have go to dev > statistics > adjust a statistic to select the sensor, date and time. So far ok.
The data is filtered immediately and i can select a row to adjust the value. But after fill in a new value and click on "adjust", the whole modal dialog is closed.

What i expected (minimal effort):

I expect that, after adjust one data value, i came back to the previous modal dialog with the priviously filled in date and time. My sensor generated a data row nealy for every 5 minutes, so i have to change 288 data rows per days and need to refill the date range very often.

What i expected (more rework):

I expect that the modal were removed and the data is presented on a new page with a standard table. I have options to select a time range and an option to adjust single data rows (like now), or adjust all data rows at once within the selected time range.

**Comment by jgrumboe** [comment]
The "minimal effort" would already be great.
I'd like to add there that it would be great to have a left-arrow and right-arrow button to browse like one page (20 minutes) back and forth. That's quicker than always updating the time input fields.

**Comment by MStapelfeldt** [comment]
I'm struggling with the same problem right now. I search the sensor, click statistic, click on "Ausreißer", pick first in list, adjust -> Dialog closes. So I have to click statistic, click on "Ausreißer", pick first in list, adjust -> Dialog closes. (repeat this 20+times.) This makes this feature the most uncomfortable feature in Home assistant to use.
I can't understand why something mentation in June '22 (https://github.com/home-assistant/frontend/discussions/13039) and here isn't adjusted or fixed after such a long time. Nice to have new ways in sorting and copy & paste tiles in frontend, better to make long existing stuff work in a user-friendly way.

**Comment by Dezorian** [comment]
Same here, its really tedious to go after all items one by one. I'd like to adjust a range to a certain number for this sensor.
